repositories/src/test/scala/cromwell/pipeline/datastorage/dao/repository/utils/TestRunUtils.scala

```diff
@@ -19,7 +19,7 @@ object TestRunUtils {
   def getDummyRun(
     runId: RunId = getDummyRunId,
     projectId: ProjectId = getDummyProjectId,
-    projectVersion: String = s"version-$randomUuidStr",
+    projectVersion: String = "v1.0.0",
```

Comment:
What's wrong with the random id?

Reply:
Because format "version-338ba0ea-8af2-11eb-8dcd-0242ac130003" is wrong.
It must be "v1.0.0"

Reply:
Got it

Reply:
Does it even matter? I mean it's a dummy run with some dummy version. We can set it to "v7.7.7" and it should not affect anything. I don't mind changing it to "v0.0.0" but I just wonder why should we care.

Reply:
Didn't see it's part of the test, ok then